xml/art_modules.xml

```diff
@@ -1313,7 +1313,7 @@
      Note that you should never delete the <guimenu>Basesystem
       Module</guimenu>. It is also not recommended to delete the
      <phrase os="sles"><guimenu>Server Applications
-      Module</guimenu></phrase><phrase os="sled">, <guimenu>&slewe;</guimenu>
+      Module</guimenu>,</phrase><phrase os="sled"><guimenu>&slewe;</guimenu>
```

Comment:
There must be no comma. Both the old and the fixed versions are incorrect. I'll fix this before merging.

Reply:
![image](https://github.com/SUSE/doc-sle/assets/98878014/0373af0b-cda2-489c-8720-4e4699430bfd)

Reply:
Yes, the old version was incorrect. Your fix however would break SLES. It would become:
> It is also not recommended to delete the _Server Applications Module_,.

Note the comma and the comma before the period.

Reply:
Anyway, I fixed it, this is nothing to worry about.